dts/arm/st/wl/stm32wl54Xc.dtsi

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2021 Thomas Stranger
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <mem.h>
+#include <st/wl/stm32wl55.dtsi>
```

Comment:
As stm32wlx5 socs differ from stm32wlx4 socs only with additional support for lora modulation a common include file is used.
Do you think there might any dt differences between this socs in the future, making it preferable including  a <st/wl/stm32wl54.dtsi> file instead?

Reply:
Let's see if @ABOSTM has more info, but I'm fine to go as is and update later if need be.